setup.py

```diff
@@ -26,7 +26,7 @@ def load_requirements(*requirements_paths):
     """
     requirements = set()
     for path in requirements_paths:
-        with open(path) as reqs:
+        with open(os.path.expanduser(path)) as reqs:
```

Comment:
Why was this needed?

Reply:
This was one of the things I used during debugging, will remove it